ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -77,7 +247,91 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
           done();
         });
       };
+      var vocabularyComponentRendered = React.addons.TestUtils.
+        renderIntoDocument(
+          <VocabularyComponent
+            vocabulary={vocabulary}
+            terms={vocabulary.terms}
+            reportError={reportError}
+            addTerm={addTerm}
+            repoSlug="repo"
+            ref={afterMount}
+          />
+        );
+      // check term title
+      var listOfLinks = React.addons.TestUtils.
```

Comment:
Please move this code inside `afterMount` and use `component` instead of `vocabularyComponentRendered`. This will ensure that they take place after the component has been mounted